recipes/openssh/files/init

```diff
@@ -1,6 +1,8 @@
 #! /bin/sh
 set -e
 
+HOST_KEY_TYPES="rsa dsa"
```

Comment:
i think that is going to be confusing. leave it blank, then you would end up with a noisy failure if somehow that list was not populated, instead of silently missing some key types. Or set it to "must-be-configured-by-recipes" :)